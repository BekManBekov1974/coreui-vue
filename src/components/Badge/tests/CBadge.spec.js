import { mount } from '@vue/test-utils'
import Component from "../CBadge";

const ComponentName = 'CBadge'
const wrapper = mount(Component)

describe(`${ComponentName} .vue`, () => {
  it('has a name', () => {
    expect(Component.name).toMatch(ComponentName)
  })
  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
});
