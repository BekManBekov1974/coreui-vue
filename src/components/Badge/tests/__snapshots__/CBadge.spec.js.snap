// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CBadge .vue renders correctly 1`] = `
<span
  class="badge badge-secondary"
/>
`;
